src/App.vue

```diff
@@ -7,17 +7,14 @@ import type { FilterSet } from './components/AddTags.vue';
 
 let loaded = ref(false);
 let gf = ref<GF | null>(null);
+window.gf = gf;
```

Comment:
Can remove this if you'd like. Does make debugging easier from browser console though.